canopen/node/local.py

```diff
@@ -127,3 +138,32 @@ def _find_object(self, index, subindex):
                 raise SdoAbortedError(0x06090011)
             obj = obj[subindex]
         return obj
+
+    def _nmt_state_changed(self, old_state, new_state):
+        if new_state == "OPERATIONAL":
+            for pdo in self.tpdo.map.values():
+                if pdo.enabled:
```

Comment:
Maybe a matter of taste, but I would prefer an inverted check and a `continue` here.  Makes for less indentation and doesn't stretch the else branch possibly out of sight.

Reply:
And this will generate warnings for any TPDO not configured as periodic, right?  Those should probably rather be skipped silently.  Have a look at the `is_periodic` property, which could be used for filtering.